client/src/main/java/com/alibaba/nacos/client/utils/EnvUtil.java

```diff
@@ -30,45 +28,42 @@
     
     public static final Logger LOGGER = LogUtils.logger(EnvUtil.class);
     
-    public static void setSelfEnv(Map<String, List<String>> headers) {
+    public static void setSelfEnv(Header headers) {
```

Comment:
Is these method refactor correctly? I see the input is come from httpclient.

Reply:
Correct, the original input is also from the http client, this method is called by the ServerListManager#getApacheServerList method